### ISSUE TITLE
Add error messages when scanning parameters

### ISSUE THREAD

**Issue by ire4ever1190** [ire4ever1190/dimscmd#9]
Return an error message to the user if they entered a command incorrectly e.g.
```
cmd.addChat("square") do (num: int):
    # square the number and send back
```
> user: $$square seven
> bot: Expected number but got "seven"

**Comment by ire4ever1190** [comment]
Easiest solution would just be to chuck the scanning into a try catch block for `ScannerError` and then just return the exception message

**Comment by GabrielLasso** [comment]
I just deployed a bot and it died as soon as an user called a command without its params. I create a try/expect block, but it'd be nice if the handleMessage had just returned `false` and the bot didn't die

**Comment by ire4ever1190** [comment]
@GabrielLasso sorry about that
That issue is fixed in c6b9927 which correctly catches errors and returns false if an error occurred because a param was missing

In other news, I'll be closing this issue now since dimscmd now returns error messages. I'll start working allowing the user to define their own error messages which will come in the update after next

**Comment by GabrielLasso** [comment]
Thanks :)
